server/src/service_manager.rs

```diff
@@ -2,17 +2,29 @@
 
 use std::{collections::HashMap, sync::Arc};
 
-use zbus::zvariant::OwnedObjectPath;
+use zbus::{zvariant::OwnedObjectPath, Connection};
```

Comment:
In the whole codebase, we only import items if they are used more than two times in the code. Otherwise it just makes it difficult to see where that type comes from.

Reply:
Understood, thanks!